requirements.txt

```diff
@@ -5,3 +5,6 @@ en_core_web_lg==3.5.0
 pdfplumber==0.9.0
 pytest==7.3.2
 openai==0.27.8
+python-multipart==0.0.6
```

Comment:
where do you use this library?

Reply:
when I was searching how to do that task with fastAPI it was required to be installed to handle multipart/form-data request but yes it was not used!